oak_functions/loader/src/lookup.rs

```diff
@@ -163,3 +178,36 @@ pub fn parse_lookup_entries<B: prost::bytes::Buf>(
     }
     Ok(entries)
 }
+
+async fn send_request<C>(client: &Client<C, Body>, request: Request<Body>) -> anyhow::Result<Bytes>
+where
+    C: Connect + Clone + Send + Sync + 'static,
+{
+    let response = client
+        .request(request)
+        .await
+        .context("could not execute request")?;
+    hyper::body::to_bytes(response.into_body())
+        .await
+        .context("could not read response body")
+}
+
+/// Gets a service account access token from the GCP metadata service.
+async fn get_access_token() -> anyhow::Result<String> {
+    let client = Client::new();
+    let request = Request::builder()
+        .method("GET")
+        .uri("http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token")
```

Comment:
Could you add a reference to where this is taken?

Reply:
Done.